yt/frontends/nc4_cm1/data_structures.py

```diff
@@ -87,13 +87,6 @@ def __init__(
         )
         self.storage_filename = storage_filename
 
-    def _setup_coordinate_handler(self):
```

Comment:
The refactor I'm doing in this PR makes `_x_pairs` and `_y_pairs` unused, so this method doesn't do anything anymore. I also cannot see a simple way to emulate this behaviour within my refactor, mostly because this method was violating the responsibility separation principle.
As a result, this is a breaking change for nc4_cm1 users, *but* there is a workaround since plot window objects recently gained axis-manipulation API (`flip_horizontal_axis`, `flip_vertical_axis` and `swap_axes`) which should allow user to reproduce plots, albeit at the cost of an additional explicit function call.

ping @keltonhalbert for information